merlin-server/constraints-dsl-compiler/src/libs/constraints-edsl-fluent-api.ts

```diff
@@ -122,6 +122,14 @@ export class Windows {
   /**
    * Invert all the windows produced this.
    */
```

Comment:
I think this comment is supposed to be attached to the `not` method below.

Reply:
I updated all the doc comments in the next commit, I think you must be viewing commit 2

Reply:
Ah, I was